firebase_admin/auth.py

```diff
@@ -363,7 +366,7 @@ def get_users(identifiers, app=None):
 
     Returns:
         GetUsersResult: A ``GetUsersResult`` instance corresponding to the
-            specified identifiers.
+        specified identifiers.
```

Comment:
Thanks. I think line 362 needs to be "A list of ``UserIdentifier`` instances...."

Reply:
Done. (It *used* to be Identifier, but then we changed it. I must have missed updating it here.)